browser-test/src/applicant/northstar_ineligible.test.ts

```diff
@@ -67,12 +67,8 @@ test.describe('North Star Ineligible Page Tests', {tag: ['@northstar']}, () => {
     })
 
     await test.step('Expect ineligible page part 1', async () => {
-      expect(await page.innerText('html')).toContain(
-        'Based on your responses to the following questions, you may not qualify for the ' +
-          programName,
-      )
-
-      expect(await page.textContent('html')).toContain(questionText)
+      await applicantQuestions.expectIneligiblePage(/* northStar= */ true)
+      await expect(page.locator('text="' + questionText + '"')).toBeVisible()
```

Comment:
(Optional) I don't know enough about playwright to know if it's preferred, but FYI there is a page. getByText() which *I* always prefer :) (same comment for other text= locators)